ocaml/xapi/extauth_plugin_ADpbis.ml

```diff
@@ -1052,6 +1052,12 @@ module AuthADlw : Auth_signature.AUTH_MODULE = struct
   let on_disable config_params =
     (* but in the ldap plugin, we should 'leave the AD/kerberos domain', i.e. we should *)
     (* (1) remove the machine account from the kerberos realm, (2) remove the keytab locally *)
+
+    (* Clean pbis cache as we are disabling external auth *)
+    ( try pbis_common "/opt/pbis/bin/ad-cache" ["--delete-all"] |> ignore
+      with _ -> debug "Got error cleaning pbis cache, ignore the error"
```

Comment:
The exception should be logged to help diagnose the problem.

Reply:
> The exception should be logged to help diagnose the problem.

I ignore the exception by intention as we do not really care about the error, (and ignored)
this is best to have clean.
But I have updated the code anyway as it always better to log errors.